packages/core/src/client.test.ts

```diff
@@ -1401,7 +1402,12 @@ describe('Client', () => {
 
     const fetch = mockFetch(200, {});
     const client = new MedplumClient({ fetch });
-    const promise = client.createBinary('Hello world', undefined, ContentType.TEXT, onProgress);
```

Comment:
Oh yeah, one last ask:

Can you add a separate test for the string version too?

My last nit, otherwise we'll get this in after tests pass on CI 👍

Reply:
ah yes, missed that!